treeherder/webapp/api/jobs.py

```diff
@@ -57,6 +57,8 @@ class JobFilter(django_filters.FilterSet):
     platform = django_filters.CharFilter(field_name='machine_platform__platform')
     ref_data_name = django_filters.CharFilter(field_name='signature__name')
     signature = django_filters.CharFilter(field_name='signature__signature')
+    task_id = django_filters.CharFilter(field_name='taskcluster_metadata__task_id')
+    retry_id = django_filters.NumberFilter(field_name='taskcluster_metadata__retry_id')
```

Comment:
I think this solves [this bug's](https://bugzilla.mozilla.org/show_bug.cgi?id=1605426) request :)

Reply:
It would, yeah.  In fact, this bug is a "see also" from that bug.  :)